cymetric/convenient_interface.py

```diff
@@ -0,0 +1,121 @@
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+
+def GetTransactionTimeSeries(db, sender='All', receiver='All', *xargs):
+
+  
+  nuc_list = []
+  for inx, nuc in enumerate(xargs):
```

Comment:
Also, `inx` is never used and so there is no need to enumerate.  You should just iterate through `xargs`: `for nuc in xargs`